commons/src/main/java/com/powsybl/commons/reporter/LoggerReporter.java

```diff
@@ -56,8 +56,8 @@ public LoggerReporter createChild(String taskKey, String defaultName, Map<String
     @Override
     public void addTaskValue(String key, Object value) {
         Objects.requireNonNull(value);
-        if (!value.getClass().isPrimitive()) {
-            throw new PowsyblException("Logger reporter expects only primitive values (value is an instance of " + value.getClass() + ")");
+        if (!(value instanceof Float || value instanceof Double || value instanceof Integer || value instanceof Long || value instanceof String)) {
+            throw new PowsyblException("Logger reporter expects only primitive or String values (value is an instance of " + value.getClass() + ")");
```

Comment:
what about byte, short, boolean, char ? Why this restriction instead of relying on Object.toString() ?

Reply:
Yeah I liked relying on Object.toString(), but then I needed json serialization/deserialization.

Reply:
I added boolean on changed the exception message to be more accurate.